libs/s25main/ingameWindows/iwTempleBuilding.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "IngameWindow.h"
+
+class nobUsual;
+class GameWorldView;
+class GameCommandFactory;
+
+class iwTempleBuilding : public IngameWindow
```

Comment:
Its a derived class now

Reply:
Perfect, much smaller class now!

BTW: Please don't "resolve" conversations that aren't just applied code as suggested. Otherwise it is hard to find your answers